SitumWayfinding/Classes/Positioning/Navigation/IndicationsViewController.swift

```diff
@@ -17,6 +17,8 @@ class IndicationsViewController: UIViewController {
     @IBOutlet weak var nextIndicationLabel: UILabel!
     @IBOutlet weak var nextIndicationImage: UIImageView!
     
+    let indicationViewCornerRadius = 5.0
```

Comment:
Vexo que esta propiedade esta escrita aquí e se lle pasa ao método `UIUtils.roundCorners` (que agora mesmo recibe un Double. Tamén esta definida noutra clase como floorSelectorCornerRadius = 10. Eu o que  faría aquí sería xerar un Enum con valores semánticos estilo default = 5.0 e big = 10.0 e pasarlle este enum a función roundCorners en vez de un número. Desta maneira todos estos valores están agrupados nun so sitio, ao pasarlle un enum fas que cando usas esto os estilos que se aplican sexan máis consistentes. E ademáis creo que queda moito máis semántico sen ter que definir este tipo de variables.